MoviesStoreApp/MoviesStoreMVVM/CoreData/CoreDataService.swift

```diff
@@ -0,0 +1,187 @@
+// CoreDataService.swift
+// Copyright © Aleksandr Shchepelin. All rights reserved.
+
+import CoreData
+
+/// Сервис по обрабокте данных в CoreData
+final class CoreDataService: CoreDataServiceProtocol {
+    // MARK: - Public Properties
+
+    lazy var managedContext: NSManagedObjectContext = self.storeContainer.viewContext
```

Comment:
```suggestion
  private  lazy var managedContext: NSManagedObjectContext = self.storeContainer.viewContext
```
снаружи не используешь